Mathlib/Combinatorics/SimpleGraph/Basic.lean

```diff
@@ -1299,23 +1349,69 @@ theorem map_adj (f : V ↪ W) (G : SimpleGraph V) (u v : W) :
   Iff.rfl
 #align simple_graph.map_adj SimpleGraph.map_adj
 
+theorem map_adj_apply {G : SimpleGraph V} {f : V ↪ W} {a b : V} :
+    (G.map f).Adj (f a) (f b) ↔ G.Adj a b := by simp
+#align simple_graph.map_adj_apply SimpleGraph.map_adj_apply
+
+@[simp]
+theorem map_id : G.map (Function.Embedding.refl _) = G :=
+  SimpleGraph.ext _ _ <| Relation.map_id_id _
+#align simple_graph.map_id SimpleGraph.map_id
+
+@[simp]
+theorem map_map (f : V ↪ W) (g : W ↪ X) : (G.map f).map g = G.map (f.trans g) :=
+  SimpleGraph.ext _ _ <| Relation.map_map _ _ _ _ _
+#align simple_graph.map_map SimpleGraph.map_map
+
```

Comment:
I assume moving these above `map_monotone` was deliberate?

Reply:
Yes, to match the `comap` lemmas. I missed that in the Lean 3 PR.